crates/config/src/sections/experimental.rs

```diff
@@ -75,6 +76,10 @@ pub struct ExperimentalConfig {
     /// Disabled by default
     #[serde(skip_serializing_if = "Option::is_none")]
     pub inactive_session_expiration: Option<InactiveSessionExpirationConfig>,
+
+    /// Experimental feature to show a plan management tab and iframe
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub plan_management_iframe_uri: Option<Url>,
```

Comment:
Not convinced that this *has* to be a URL? Like if we want to have a relative link here, it won't get accepted?

Reply:
> I'm indeed thinking of just passing it as as string without validation

I've converted into a `String` with some additional doc comments in https://github.com/element-hq/matrix-authentication-service/commit/d6dd647f79258d2ff951a1d5752e37b141101cfd